.travis.yml

```diff
@@ -13,13 +13,23 @@ matrix:
   include:
     - os: linux
       env: CUDA=8.0.61-1
-    - os: osx
+    - os: linux
+      env: QT_BASE=59
     - os: linux
       env: STATIC_ANALYSIS=ON
+    - os: osx
+      env: QT_BASE=latest
 
 before_install:
-   - source script/travis/before_script.sh
+  - source script/travis/before_script.sh
+
+install:
+  - if [ "$QT_BASE" = "59" ]; then sudo apt-get install -qq qt59base; source /opt/qt59/bin/qt59-env.sh; fi
```

Comment:
We need to disable CUDA installation for `QT_BASE=59` job. It fails actually. Let's separate CUDA and QT for now.